src/base/rss/rss_feed.cpp

```diff
@@ -195,7 +194,7 @@ bool Feed::hasError() const
 void Feed::handleDownloadFinished(const Net::DownloadResult &result)
 {
     if (result.status == Net::DownloadStatus::Success) {
-        qDebug() << "Successfully downloaded RSS feed at" << result.url;
+        LogMsg(tr("RSS feed at '%1' is successfully downloaded. Start to parse it."));
```

Comment:
You have rushed. You forgot the substitution for `%1`.

Reply:
Oops... Fixed.